src/controller/org.controller/index.js

```diff
@@ -322,14 +317,13 @@ router.get('/org/:shortname/users',
   /*
   #swagger.tags = ['Users']
   #swagger.operationId = 'userOrgAll'
-  #swagger.summary = "Retrieve all user records specified by the short name of the organization"
+  #swagger.summary = "Retrieves all user information in an organization for the speicifed short name (accessible to all users)"
```

Comment:
typo in "speicified". Maybe reword to "Retrieves all users for the organization with the specified short name"